content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
+
+### Project
+
+In each workspace, you can create several projects. A project typically represents an application (a product or service) your organization is offering. You typically have one [Prisma schema](/orm/prisma-schema/) per project.
+
+In each project, you can:
+
+- view and manage all environments in that project.
+
+The number of project you can create in a workspace depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in that workspace:
+
+Here's an overview of the number of projects per workspace on each plan:
+
+|              | **Starter** | **Pro** | **Business** | **Enterprise** |
+| ------------ | ----------- | ------- | ------------ | -------------- |
+| **Projects** | 5           | 10      | 15           | Custom         |
+
+### Environment
+
+An environment is an isolated space to provision PDP products for a given project. Environments typically map to development stages, like `dev`, `staging` or `prod`.
+
+In each environment, you can:
+
+- enable, disable and manage PDP products (Pulse, Accelerate, ...).
+- for **Accelerate**:
+  - set your database connection string.
+  - generate API keys.
+  - change the connection pool size.
+  - enable static IPs.
+- for **Pulse**:
+  - set your database connection string (of a [Pulse-compatible database](/pulse/database-setup)).
+  - configure the _region_ where Pulse is running.
+  - configure database replication.
+  - generate API keys.
+
+The number of environments you can create in a project depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in your workspace:
+
+Here's an overview of the number of environments per project on each plan:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | ----------- | ----------- | ------------ | -------------- |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+## Billing
+
+The [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) you select in your workspace determins how many projects and environments you can create in that workspace:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | :---------- | ----------- | ------------ | -------------- |
+| **Projects**     | 5           | 10          | 15           | Custom         |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+### Per-workspace billing
+
+Billing in PDP is done on a per-workspace basis:
+
+- A pricing plan is selected per workspace. That means, your PDP account can have multiple workspaces where each workspace uses a different pricing plan.
+- A payment method is selected per workspace. That means, your PDP account can have multiple workspaces where each workspace has a different payment method.
```

Comment:
```suggestion
- A payment method is selected per workspace. That means, your user can belong to multiple workspaces where each workspace has a different payment method.
```

Reply:
updated to: 

- A payment method is selected per workspace. That means, a user account can belong to multiple workspaces where each workspace has a different payment method.